templates/solidity-frameworks/hardhat/packages/hardhat/hardhat.config.ts.template.mjs

```diff
@@ -41,6 +41,7 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
+    ${networks},
```

Comment:
${networks[0] ? \`${networks[0]},\` : ""}  helps. 
Note: `networks` is array so I think to better to be explicit